docs/book/src/02_getting_started.md

```diff
@@ -23,7 +23,12 @@ cargo init leptos-tutorial
 `cd` into your new `leptos-tutorial` project and add `leptos` as a dependency
 
 ```bash
-cargo add leptos --features=csr,nightly # or just csr if you're using stable Rust
+cargo add leptos --features=csr,nightly
+```
+
+Or you can leave off `nighly` if you're using stable Rust
+```bash
+cargo add leptos --features=csr
```

Comment:
The comment in the previous version breaks copy-paste. If you're not familiar with the command line it's no clear why its broken.